internal/grpc/services/appregistry/appregistry_test.go

```diff
@@ -168,6 +186,45 @@ func Test_GetAppProviders(t *testing.T) {
 		},
 	}
 
+	mimeTypes := map[string]map[string]string{
+		"text/json": {
+			"extension":   "json",
+			"name":        "JSON File",
+			"icon":        "https://example.org/icons&file=json.png",
+			"default_app": "some Address",
+		},
+		"text/xml": {
+			"extension":   "xml",
+			"name":        "XML File",
+			"icon":        "https://example.org/icons&file=xml.png",
+			"default_app": "some Address",
+		},
+		"application/vnd.openxmlformats-officedocument.wordprocessingml.document": {
```

Comment:
I propose to keep the test examples smaller and remove all except JSON and XML. Then we should populate the example configuration file with some sensible defaults.

Reply:
As discussed, the mime types in the test are the minimum necessary to pass the tests.
For the example config, I've added all the extensions you mentioned except the root as it does not have a defined mime type.